src/Compilers/CSharp/Portable/Symbols/Attributes/RetargetingAttributeData.cs

```diff
@@ -43,7 +43,7 @@ internal sealed class RetargetingAttributeData : CSharpAttributeData
 
         public override SyntaxReference? ApplicationSyntaxReference => null;
 
-        [MemberNotNullWhen(true, nameof(AttributeClass), nameof(AttributeConstructor))]
```

Comment:
>nameof(AttributeClass) [](http://example.com/codeflow?start=33&length=22)

It looks like removal of ```AttributeClass``` is not necessary.

Reply:
The attribute adds no information for `AttributeClass`, as that API is already declared as non-nullable.

Reply:
> The attribute adds no information for AttributeClass, as that API is already declared as non-nullable.

That was incorrect annotation and it is already fixed in main.